test/integration/shared/get-metrics.test.ts

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios'
```

Comment:
question: won't these get maybe a little cluttered containing both Inbound and Outbound tests?

Reply:
eh. nvm. It'll probably be fine.

Reply:
I would like to keep the same structure for integration tests so the output on console log is much more consistent